docs/concepts/kuery.asciidoc

```diff
@@ -144,6 +144,8 @@ but in some cases you might need to search on dates. Include the date range in q
 @timestamp < "2021"
 -------------------
 
+KQL also supports queries such as `now-1h/d`. Check out the
```

Comment:
Can we actually provide an _example_ of such a query? We provide three example date queries above this text, it would make sense to provide an example of this query, too, using the `@timestamp` field.